README.md

```diff
@@ -0,0 +1 @@
+# NearSnap
```

Comment:
could you add short description of what each package does and some info how to use it?